pkg/apis/core/validation/validation.go

```diff
@@ -3332,6 +3315,12 @@ func ValidatePodSecurityContext(securityContext *core.PodSecurityContext, spec *
 				allErrs = append(allErrs, field.Invalid(fldPath.Child("shareProcessNamespace"), *securityContext.ShareProcessNamespace, "ShareProcessNamespace and HostPID cannot both be enabled"))
 			}
 		}
+
+		if utilfeature.DefaultFeatureGate.Enabled(features.Sysctls) {
```

Comment:
if the feature gate is enabled, we must validate the field, but if the feature gate is disabled, we should prevent usage of the field.  see similar example above for ShareProcessNamespace.

Reply:
Done.